.github/workflows/automerge.yml

```
@@ -0,0 +1,36 @@
+        id: check_approvals
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const { owner, repo, number: pull_number } = context.issue;
+            const reviews = await github.rest.pulls.listReviews({
+              owner,
+              repo,
+              pull_number,
+            });
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)